structuralalignment/superposition/theseus.py

```diff
@@ -107,18 +64,14 @@ def __init__(self, max_iterations: int = 32):
         self.alignment_file = "theseus.aln"
         self.alignment_app = "muscle"
 
-    def _create_pdb(self, structures):
+    def __create_pdb(self, structures):
```

Comment:
We use `_single_leading_underscores` for "private" methods (by convention). `__double_leading_underscores` have a different meaning that have to do with subclasses and class attributes, beyond the scope of this PR!